### ISSUE TITLE
Unknown props 'mobile' warning

### ISSUE THREAD

**Issue by pwhipp** [vlad-ignatov/react-numeric-input#20]
This warning when using react-numeric-input 2.0.4 in react 15.2.1 does not seem to break anything but it is annoying:

```
Warning: Unknown props `mobile`, `onSelectStart` on <input> tag. Remove these props from the element. For details, see https://fb.me/react-unknown-prop
    in input (created by NumericInput)
    in span (created by NumericInput)
    in NumericInput (created by NumberControl)
    ...
```

**Comment by freeranger** [comment]
Already reported here: https://github.com/vlad-ignatov/react-numeric-input/issues/16

**Comment by vlad-ignatov** [comment]
This is fixed v2.0.5 which adds compatibility with React 15.2+
